### PR TITLE
Create 0000-01-02-flypalindrome.md

### PR DESCRIPTION
**Pull request by flypalindrome**
Hey, I'm gonna commit new file. Now things aren't all clear, but it will get better soon.

### CODE REVIEW

File: _posts/0000-01-02-flypalindrome.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (flypalindrome) [inline _posts/0000-01-02-flypalindrome.md:5]: Hi, still don't know anything.

Comment (lkjali) [inline _posts/0000-01-02-flypalindrome.md:5]: I'm Ali

Comment (lkjali) [inline _posts/0000-01-02-flypalindrome.md:5]: finally

**Review by lkjali — APPROVED** [review]
hihihi

**Comment by flypalindrome** [comment]
Um... Finally it's time to close with comment.

**Comment by flypalindrome** [comment]
abcabcabc

File: _posts/0000-01-02-flypalindrome.md (hunk 1 above)
Comment (flypalindrome) [inline _posts/0000-01-02-flypalindrome.md:5]: erwqeeeeeewerrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrroqupiuqwrpoiquweropiuapoifdsupoidasf

**Comment by flypalindrome** [comment]
dfgrtrt

**Pull request merged**